pkg/storage/trb_request_feedback.go

```diff
@@ -42,6 +49,7 @@ func (s *Store) CreateTRBRequestFeedback(ctx context.Context, feedback *models.T
 			:copy_trb_mailbox,
 			:notify_eua_ids,
 			:action,
+			:created_at,
```

Comment:
Previously any created feedback row would default to the current timestamp, which isn't great for testing purposes.